__tests__/app.test.js

```diff
@@ -191,6 +191,16 @@ describe('GET: /api/reviews/:review_id/comments', () => {
             .get('/api/reviews/2/comments')
             .expect(200)
     })
+    it('responds with status code 200 and a blank array if given a valid id with no reviews', () => {
+        return request(app)
+            .get('/api/reviews/1/comments')
+            .expect(200)
+            .then(( res ) => {
+                const body = res.body.comments
+                expect(body).toBeInstanceOf(Array)
+
```

Comment:
Make sure you specify that its really an empty array. Totally okay to hard-code an empty array and assert that the returned one is equal to it.